extensions/wikia/ImageReview/ImageReviewImageStatus.php

```diff
@@ -34,6 +34,12 @@ function efImageReviewDisplayStatus( ImagePage $imagePage, &$html ) {
 	if ( in_array( $wgCityId, $wgImageReviewTestCommunities ) ) {
 		$reviews =
 			fetchReviewHistoryFromService( $wgCityId, $imagePage->getTitle() );
+
+		// TODO: temporary solution, remove it as soon as image review history is migrated to the new tool
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)